documentation/release_notes/v4.0.md

```diff
@@ -5,4 +5,4 @@ This is major new release, providing internal changes to the sqw/dnd objects, ne
 ## Critical changes
  - `DnD` objects are fully refactored and constructed from `axes_block` class (property `axes`), `aProjection` class (property   `proj`) and three datasets, namely `s,e` and `npix` containing signal, error and number of pixels contributed into a bin, as before. `axes_block` defines binning of these datasets and their physical meaning, while projection defines the transformation from pixels coordinate system to the coordinate system, defined by `axes_block`. Old interface (`iax,pax,iint,p etc.`) left for compatibility with existing algorithms and set as read-only interface.
  - `data_sqw_dnd` class (property `data` of `sqw` object) have been removed from `sqw` object and replaced by appropriate dimension `dnd` object. Pixels data (`PixelData` class), which were the property of `data_sqw_dnd` class are now independent property of `sqw` object (property `pix`). Old and refactored `data_sqw_dnd` class left for compatibility with old `sqw` files, allowing the loading of old data from `.mat` files. 
- 
\ No newline at end of file
+  - `multifit` and `tobyfit` are now able to be run in parallel through the `hpc_config` `parallel_multifit` setting. 
```

Comment:
Appreciating that these are release notes rather than a tutorial, but some indication of how to activate this parallel operation would be helpful - reference to instructions elsewhere, actual info, whatever.

Reply:
It is true that one of the things on my to-do list is to write proper docs, but I think this is best kept out of the release notes. I will write the docs up in the fitting section of the main docs.